setup.py

```diff
@@ -32,6 +32,9 @@
 import os
 import pkgutil
 import sys
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'lib'))
```

Comment:
apparently not...

maybe this is taken care of in the package that gets uploaded to PyPI using vsc-install, but to install from the repo (or GH source tarball), you need this there...

@stdweird ?